vigranumpy/src/core/axistags.cxx

```diff
@@ -646,7 +650,10 @@ void defineAxisTags()
              "contains the given axistype.\n")
         .def("compatible", &AxisInfo::compatible,
              "\naxisinfo1.compatible(axisinfo2) yields True when the two axisinfo objects "
```

Comment:
```suggestion
             "\naxisinfo1.compatible(axisinfo2) yields True when the two axisinfo objects\n"
```

just because the current rendering is a bit ugly:

```
 |  compatible(...)
 |      axisinfo1.compatible(axisinfo2) yields True when the two axisinfo objects have the same keys and types, or either of the two is 'unknown'.
 |      Note that, as this does not compare axis resolutions or units,
 |      there may still be mismatch or incompatibility in that regard.
```